docs/shared/guides/global-nx.md

```diff
@@ -73,6 +73,12 @@ pnpm list --global nx
 {% /tab %}
 {% /tabs %}
 
+{% callout type="note" title="Older Global Installations" %}
+
+In prior versions, Nx could be installed globally via `@nrwl/cli` or `@nrwl/tao`. If you are seeing these warnings but cannot find other global installations of Nx via the above commands, you should look for these packages as well. In general, you should remove these and install the latest version of `nx` instead.
```

Comment:
This is fine, but I would update the remove scripts below to also remove `@nrwl/cli` and `@nrwl/tao`

Reply:
I wanted to avoid putting them in too many places, since we'd probably want to remove this note all together in v18 or so, after the rename has had enough time. I can add them there too.